products/artifactregistry/api.yaml

```diff
@@ -0,0 +1,102 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: ArtifactRegistry
+display_name: Artifact Registry
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://artifactregistry.googleapis.com/v1beta1/
```

Comment:
What happens when we upgrade to v1?  Do we have multiple version of this, or just change this string?

Reply:
If there are multiple APIs we try to keep multiple `versions` specified here. That will generate [terraform-provider-google-beta](https://github.com/terraform-providers/terraform-provider-google-beta) which will call into the beta API and [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) which will call the v1 endpoints. In the future if fields are added to the beta API that haven't made it to v1 we can support those.